srcpkgs/emilua/template

```diff
@@ -0,0 +1,40 @@
+# Template file for 'emilua'
+pkgname=emilua
+version=0.4.3
+revision=1
+archs="x86_64"
+build_style=meson
+build_wrksrc="${pkgname}-v${version}"
+hostmakedepends="LuaJIT-devel boost-devel fmt-devel openssl-devel ncurses-devel serd-devel sord-devel liburing-devel libcap-devel"
+makedepends="meson boost re2c gawk gperf xxd ruby-asciidoctor pkg-config"
```

Comment:
`hostmakedepends` and `makedepends` are the opposite.

Reply:
Oh, my mistake.
I will update the template.